index.html

```diff
@@ -961,13 +961,26 @@ <h2><dfn>Coalesced events</dfn></h2>
                 of the pointer movement.</figcaption>
             </figure>
 
-            <p>A <a>PointerEvent</a> has an associated <dfn>coalesced event list</dfn> (a list of
+            <p>A <a>PointerEvent</a> has an associated <dfn>coalesced events list</dfn> (a list of
             zero or more <code>PointerEvent</code>s). If this event is a <code>pointermove</code>
             or <code>pointerrawupdate</code> event, the list is a sequence of all <code>PointerEvent</code>s
             that were coalesced into this event; otherwise it is an empty list.</p>
 
-            <p>The events in the coalesced event list will have increasing
-            <a href="https://dom.spec.whatwg.org/#dom-event-timestamp"><code>timeStamps</code></a>
+            <p>When the <code>getCoalescedEvents()</code> method is called, user agents MUST apply the following
```

Comment:
I think it's easier if we "finalize" both lists at the moment the event was dispatched, instead of the moment when `get*Events()` is called.  The target is known at dispatch time, so do we need to delay the target fixup step at all?

If there is a reason to delay it (performance?), then the first call to any of `get*Events()` can finalize both lists to avoid complications.

Reply:
I don't think the spec should say when this happens, and just say that the coalesced events will have the same target.

However, I think an implementation is free to do this whenever it chooses. It shouldn't be developer observable when this happens.

Reply:
Nevermind, per discussion it is observable when the target changes so we should retarget here.